src/opnsense/scripts/firmware/bogons.sh

```diff
@@ -58,7 +58,7 @@ LINES_V4=`wc -l ${WORKDIR}/fullbogons-ipv4.txt | awk '{ print $1 }'`
 if [ $ENTRIES_MAX -gt $((2*ENTRIES_TOT-${ENTRIES_V4:-0}+LINES_V4)) ]; then
     # private and pseudo-private networks will be excluded
     # as they are being operated by a separate GUI option
-    egrep -v "^100.64.0.0/10|^192.168.0.0/16|^172.16.0.0/12|^10.0.0.0/8" ${WORKDIR}/fullbogons-ipv4.txt > ${DESTDIR}/bogons
+    egrep -v "^169.254.0.0/16|^100.64.0.0/10|^192.168.0.0/16|^172.16.0.0/12|^10.0.0.0/8" ${WORKDIR}/fullbogons-ipv4.txt > ${DESTDIR}/bogons
```

Comment:
once we have IPv6 figured out (see last suggestion) we can adjust this the same way in a separate PR? :)

Reply:
Good idea. So for this PR, adding 169.254.0.0/16 here is okay with you?

Reply:
Let's break this into two PRs for each address family, but both also changing their egrep -v into the method proposed for IPv6

Reply:
I reverted all IPv4 changes.